test-resources/swift/cleanup_rules/expected/TestEnum.swift

```diff
@@ -0,0 +1,17 @@
+// Copyright (c) 2022 Uber Technologies, Inc.
+// 
+// <p>Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file
+// except in compliance with the License. You may obtain a copy of the License at
+// <p>http://www.apache.org/licenses/LICENSE-2.0
+// 
+// <p>Unless required by applicable law or agreed to in writing, software distributed under the
+// License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+// express or implied. See the License for the specific language governing permissions and
+// limitations under the License.
+
+enum TestEnum {
+    case stale_flag_one
+    var isEnabled: Bool {
+        return true
+    }
+}
```

Comment:
please add new line. I wonder how did this PR make it. It should have been caught by our pre-commit rule